x-pack/plugins/licensing/public/plugin.ts

```diff
@@ -134,6 +134,7 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup, LicensingPl
     }
     return {
       refresh: this.refresh,
+      getLicense: async () => await firstValueFrom(this.license$!),
```

Comment:
I think it is worth updating docs in the Readme, right now docs say `refresh: () => Promise<ILicense> allows a plugin to enforce license retrieval.` which reads misleading

Reply:
great point! I'll update the readme